### ISSUE TITLE
✔️ 🚀 Feature Request: Better background lazyload technique

### ISSUE THREAD

**Issue by bouveng** [elementor/elementor#20479]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

The current impl. of the background lazy loading, loads a 150x150 thumbnail.
This is not an optimal or desired behavior, creating not needed http requests.

### Describe the solution you'd like

Just use the `data-e-bg-lazyload` data attribute to target the background with a `[data-e-bg-lazyload] {background-image:none!important;}` (the image wont be loaded)

Then, instead of setting the `lazyloaded` class, just remove the `data-e-bg-lazyload` data attribute and the image will load.

Please consider this instead:

JS:

```js
( () => {

	const sel = '[data-e-bg-lazyload]';
	let els = document.querySelectorAll( sel );

	const config = { rootMargin: '100px 0px 100px 0px' }

	let observer = new IntersectionObserver( ( entries, observer ) => {
		entries.forEach( entry => {
			if ( entry.isIntersecting ) {
				let el = entry.target;
				el.removeAttribute( sel );
				observer.unobserve( el );
			}
		});
	}, config );

	els.forEach( el => observer.observe( el ) );

})();
```

CSS:

```css
[data-e-bg-lazyload] {background-image:none!important;}
```

JS MIN (262 bytes), -37.3% from current:
```js
(()=>{let e="[data-e-bg-lazyload]",r=document.querySelectorAll(e),t={rootMargin:"100px 0px 100px 0px"},
o=new IntersectionObserver((r,t)=>{r.forEach(r=>{if(r.isIntersecting){let o=r.target;o.removeAttribute(e),
t.unobserve(o)}})},t);r.forEach(e=>o.observe(e))})();
```

:nerd_face:

**Comment by nicholaszein** [comment]
Hello @bouveng,

thanks for submitting your feedback! 🙏 

I've already labeled your issue and the team will be taking a look at your suggestion.

Best regards

**Comment by nicholaszein** [comment]
Hello @bouveng and @mcrowell!

We have great news! 🙌 

### 📢 We're happy to announce that this issue was resolved in Elementor Pro **v3.9.0**! 🥳 

We did not use the method suggested by @bouveng, but we did change how the lazy load engine works, which solves this issue and improves performance. But thanks anyway for your suggestion! 🙏 

✔️ Please feel free to check it out and update your plugin to the new version!

Cheers 🥂

**Comment by curlygirly1** [comment]
Hi. How to PREVENT a BG image from lazyloading? 

I have above-fold hero video and till it finally loads I want a BG image in the container to load, so it's not a gaping blackness. 
I hunted everywhere and cannot find a solution :(

Weird too is that some below fold images load before this BG image that's above fold.
